CONTRIBUTING.md

```diff
@@ -143,14 +144,17 @@ Make a PR modifying the following files with the right version:
 export const PACKAGE_VERSION = 'X.X.X'
 ```
 
+#### Github Publish
 Once the changes are merged on `main`, you can publish the current draft release via the [GitHub interface](https://github.com/meilisearch/instant-meilisearch/releases): on this page, click on `Edit` (related to the draft release) > update the description (be sure you apply [these recommendations](https://github.com/meilisearch/integration-guides/blob/main/resources/integration-release.md#writting-the-release-description)) > when you are ready, click on `Publish release`.
 
 GitHub Actions will be triggered and push the package to [npm](https://www.npmjs.com/package/@meilisearch/instant-meilisearch).
 
+#### Codesandbox update
 Once the version is available on npm, please update the instant-meilisearch version used in the different Code-Sandboxes we provide:
 
 - [Meilisearch + InstantSearch](https://codesandbox.io/s/ms-is-mese9)
-- [Meilisearch + Vue InstantSearch](https://codesandbox.io/s/ms-vue-is-1d6bi)
+- [Meilisearch + Vue 2 InstantSearch](https://codesandbox.io/s/ms-vue-is-1d6bi)
+- [Meilisearch + Vue 3 InstantSearch](https://codesandbox.io/s/ms-vue3-is-0293zk)
```

Comment:
Should not be on this PR, if you want me to open a separate PR with that change, feel free to tell me. Otherwise, if you don't see it I don't see it 🙈

Reply:
I have not seen anything